from_csv_to_json.py

```diff
@@ -0,0 +1,58 @@
+from converter_file_format import Converter
+from json import dumps
+
+class CSV_JSON(Converter):
```

Comment:
Class name is not according to pep8, you could call it CsvToJson, or introduce a new name